addons/blocks2image/addon.json

```diff
@@ -16,6 +20,20 @@
       "matches": ["projects"]
     }
   ],
+  "userstyles": [{ "url": "style.css", "matches": ["projects"] }],
   "versionAdded": "1.13.0",
-  "enabledByDefault": false
+  "enabledByDefault": false,
+  "latestUpdate": {
+    "version": "1.39.0",
+    "temporaryNotice": "The addon has been reworked. There is now a modal for saving the blocks as an image, and the image can be copied to the clipboard.",
+    "isMajor": true
+  },
+  "info": [
+    {
+      "id": "keys",
+      "type": "info",
+      "text": "You can press the enter key to copy the block to the clipbard, or press the 1 - 3 keys corresponding to the buttons."
+    }
+  ],
+  "permissions": ["clipboardWrite"]
```

Comment:
This may disable the addon for all Firefox users, even if they don't plan to use the clipboard feature. Please don't close this comment. I'm researching how we can proceed

Reply:
I mentioned per-setting permissions earlier, but I don't think any other addons would use it (https://github.com/ScratchAddons/ScratchAddons/pull/6830#discussion_r1451782282).

Reply:
In the latest version of Firefox, the clipboardWrite permission can be ignored (it is already ignored in Chrome). See #7625

Reply:
We could make it so that Scratch Addons doesn't respond to missing permissions (that aren't crucial) by disabling addons, or we could request the necessary permissions somehow.

Reply:
I created PR #7630.

Only around ~1000 users run Firefox versions lower than 127, and of course not all of them use this addon, so it's only a few users that will see this addon turn off automatically.
This addon will be positioned at the top of the settings page due to its "new" status, so it will be easy to find it.

Reply:
Solved by PR #7630